### PR TITLE
404mk3

### PR DESCRIPTION
**Pull request by bkmac511**

### CODE REVIEW

File: apps/frontend/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          <Link href="/about" passHref>
+          <a className={styles.menu_element}>O nas</a>
+          </Link>
+        </li>
+      </ul>
+
+      </div>
+
+    </div>
+
```
Comment (mtatko) [inline apps/frontend/pages/404.tsx:37]: I feel like you should work on formatting this file a little bit

**Review by mtatko — CHANGES_REQUESTED** [review]

File: apps/frontend/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <h1 className={styles.header}><b>404.</b></h1>
+
+      <a className={styles.properText}>Może szukałeś czegoś innego? Sprawdź to:</a>
+
+      <div className={styles.divlist}>
+
+      <ul className={styles.listmenu}>
+        <li>
+          <Link href="/" passHref>
+          <a className={styles.menu_element}>Strona Główna</a>
```
[the remark this answers is not in the document]
Comment (bkmac511) [reply apps/frontend/pages/404.tsx]: ```Strona główna```, i changed it already.

File: apps/frontend/components/footer/TemplateAuthors.tsx [hunk 3]
```diff
@@ -22,3 +23,4 @@ export function TemplateAuthors() {
     </>
   );
 }
+//
```
[the remark this answers is not in the document]
Comment (bkmac511) [reply apps/frontend/components/footer/TemplateAuthors.tsx]: Deleted.

File: apps/frontend/pages/404.tsx (hunk 1 above)
Comment (bkmac511) [reply apps/frontend/pages/404.tsx]: Everything is formatted now. Thx for feedback. Over and out.

**Review by mtatko — APPROVED** [review]

File: apps/frontend/pages/404.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+        <li>
+          <Link href="/news" passHref>
+          <a className={styles.menu_element}>Aktualności</a>
+          </Link>
+        </li>
+
+        <li>
+          <Link href="/about" passHref>
+          <a className={styles.menu_element}>O nas</a>
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:29]: you're mixing variable names cases `menu_element, properText`

File: apps/frontend/pages/404.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import Link from 'next/link'
+import styles from './404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.wrapper}>
+
+      <h1 className={styles.header}><b>404.</b></h1>
```
Comment (konhi) [inline apps/frontend/pages/404.tsx:8]: you still didn't use accesible headings

**Comment by mtatko** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Updated
> **zseis-zgora**	🔄 Building ([Inspect](https://vercel.com/elektronplus/zseis-zgora/BSvB48E82bYVjQ5m3aFo9RAAerR7))		Oct 2, 2022 at 7:15PM (UTC)

the broken revert commits are still present on the fork

File: apps/frontend/queries/getErrorPage.graphql [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+	page404{
+		data{
+			attributes{
+				title
+				description
+				links
+			}
+		}
+	}
+}
```
Comment (mtatko) [inline apps/frontend/queries/getErrorPage.graphql:11]: Remember to end the file with a new line - it is a good practice.

**Comment by bkmac511** [comment]
> The build passes fine - https://zseis-zgora-cqdp9v7dw-elektronplus.vercel.app. The error pages work too. image

We can't deploy it now because we have to replace normal links, with ones that we'll get from GraphQL queries.

**Comment by mtatko** [comment]
> We can't deploy it now because we have to replace normal links, with ones that we'll get from GraphQL queries.

IKR, just letting you know

**Comment by bkmac511** [comment]
> > We can't deploy it now because we have to replace normal links, with ones that we'll get from GraphQL queries.
> 
> IKR, just letting you know

I just don't know why now check fail after formatting some text

**Comment by mtatko** [comment]
> I just don't know why now check fail after formatting some text

A team member on Vercel has to authorize every deploy from the fork

**Comment by bkmac511** [comment]
> A team member on Vercel has to authorize every deploy from the fork

OK

**Comment by bkmac511** [comment]
https://zseis-zgora-ffftcumou-elektronplus.vercel.app/ the build is working

**Comment by mtatko** [comment]
what is this commit message lol (70309f)

**Pull request merged**